tests/integ/test_mxnet_train.py

```diff
@@ -52,6 +52,7 @@ def test_attach_deploy(mxnet_training_job, sagemaker_session):
         predictor.predict(data)
 
 
+@pytest.mark.continuous_testing
 def test_deploy_model(mxnet_training_job, sagemaker_session):
```

Comment:
why that one?

Reply:
I think it tests a more straightforward code path than `test_attach_deploy`, though I could be persuaded to pick the other one since our TF test already tests the "straightforward" approach.

Reply:
talked offline - will switch to `test_attach_deploy` since deploying through the estimator rather than creating a separate model object is probably more common

Reply:
It doesn't really matter, i just find this test to be more unnatural, that's why was curious about the choice process :)
Approved.